### PR TITLE
GIA-3452 - Recommend button doesn't give proper feedback when clicked

### PR DESCRIPTION
**Pull request by sandropaganotti**

### CODE REVIEW

**Comment by sandropaganotti** [comment]
Ready to be code reviewed /cc @guardian/developers-write

File: ArticleTemplates/assets/js/modules/comments.js [hunk 1]
```diff
@@ -126,12 +126,12 @@ define([
 
                 // Functions for feedback on recommend buttons
                 window.commentsRecommendIncrease = function (id, number) {
-                    var target = '#' + id + ' .comment__recommend';
+                    var target = "div[id='" + id + "'] .comment__recommend";
```
Comment (johnduffell) [inline ArticleTemplates/assets/js/modules/comments.js]: did you mean to change single quotes to double quotes?

File: test/unit/comments.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+define([
```
Comment (johnduffell) [inline test/unit/comments.js]: unit tests are awesome! =)

**Comment by johnduffell** [comment]
looks good to me! :+1:

**Comment by sandropaganotti** [comment]
Perfect, thanks ! ^_^

**Pull request merged**